service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseDataExtractionService.java

```diff
@@ -69,10 +69,11 @@ public String getHearingTime(HearingBooking hearingBooking) {
         return hearingTime;
     }
 
-    public Optional<String> getHearingDateIfHearingsOnSameDay(HearingBooking hearingBooking) {
+    public Optional<String> getHearingDate(HearingBooking hearingBooking) {
```

Comment:
You need to update the classes that use this service. NoticeOfHearingGenerationService.java for example is now breaking as it's trying to reference getHearingDateIfHearingsOnSameDay

Reply:
yeah, sorry ... before check-in this file was not changed.. I need to merge it before any commits, will keep that in mind

Reply:
np :)

Reply:
Is this change related to tech debt? If yes please link to the commit / ticket name (so we know in history why the change was introduced)

Alternatively do we agree with the change? I don't know why this is being renamed, the name was nicely telling what this method is doing, now the name is worse...

Reply:
- The branch name itself says it's a tech debt, so I feel that's good enough indication 
- There is another method getHearingTime, that also has the same logic , but the method name is still getHearingTime
so from consistency point of view , method names are fine ...  if we need to change, then we need to change both the method names 
  
thoughts?